tests/webidl/post.js

```diff
@@ -287,5 +287,14 @@ if (isMemoryGrowthAllowed) {
 
 //
 
+const fs = require('fs')
+const data = fs.readFileSync('glue.cpp', 'utf8')
+if (data.includes("int uCharArrayTest(const unsigned char* arg, int size)")) {
+	console.log('JSimplementation array argument success')
+}
+else {
+	console.log('JSimplementation array argument failure')
+}
+
```

Comment:
It would be better to test this in a more direct way than to look for a text match in the output, I think. That is, like the other tests above, to run some code that would have failed (or not even compiled) before this fix. An advantage of doing it that way is that small text output changes (say, whitespace or line breaks) would not break the test, and also, seeing running code end to end is nice.

Reply:
I'll be pushing a pared down version of the unit test since each type saw the same behaviour. The reason I originally did the test above is that it avoided throwing an error message and allowed the testing script to terminate normally. The new test will actually be running the example function and without the fix will throw an error instead of terminating. Expected error below:

> error: cannot initialize a parameter of type 'char' with an lvalue of type 'char *'
>   return self->charArrayTest(arg);
>                              ^~~
> note: passing argument to parameter 'arg' here
>   bool charArrayTest(char arg)  {

Note, in the process of rewriting the unit test for this I noticed there seems to be a similar issue with overlooking square brackets denoting arrays in the return type of functions. eg

`byte[] exampleMethod(); --> char exampleMethod() {...`

I avoided remedying that issue since this pr is specifically related to array types in argument lists, but I am fairly certain the issue stems from how the return_type variable is created around webidl_binder.py line 675 once again not being given an opportunity for the isArray attribute to be accounted for. Though since that variable is used in several places it is a little more tricky to ensure a fix does not adversely affect any other parts of the code.